docs/core/deploying/native-aot/index.md

```diff
@@ -143,10 +165,19 @@ Publishing a class library as native AOT creates a native library that exposes m
 
 The following table shows supported compilation targets.
 
+### [.NET 7](#tab/net7)
+
 | Platform | Supported architecture |
 |----------|------------------------|
 | Windows  | x64, Arm64             |
 | Linux    | x64, Arm64             |
-| macOS*   | x64, Arm64             |
 
-\* Supported starting in .NET 8.
+### [.NET 8+](#tab/net8plus)
+
+| Platform | Supported architecture |
+|----------|------------------------|
+| Windows  | x64, Arm64             |
+| Linux    | x64, Arm64             |
+| macOS    | x64, Arm64             |
```

Comment:
> @ivanpovazan @kotlarmilos @akoeplinger @steveisok How would you like to go about iOS/tvOS/etc. support? Should we document it here with an asterisk that it's experimental, or omit for now?

That only shows up on the .NET 8 tab. I removed the `*` because it's easy to miss. This makes it obvious.

Reply:
> @ivanpovazan @kotlarmilos @akoeplinger @steveisok How would you like to go about iOS/tvOS/etc. support? Should we document it here with an asterisk that it's experimental, or omit for now?

I would vote +1 to add it with an asterisk (noting it is experimental).
Speaking of which, what about linux bionic?